.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
I don't know. Tackling all those checks is pretty hard. For e.g. `no-self-use` arises in so many functions., `too-many-locals` is also bound to happen sooner or later, `duplicate-code` arises even if there is only single line match. I don't think I have disabled any crucial checks, though (they don't seem as important as others).

I tried removing many of them but let me do it once again. Then we can talk about the remaining once..